code/game/objects/items/grenades/atmos_grenades.dm

```diff
@@ -0,0 +1,95 @@
+/obj/item/grenade/gas_crystal
+	desc = "Some kind of crystal, this shouldn't spawn."
+	name = "Gas Crystal"
+	icon = 'icons/obj/grenade.dmi'
+	icon_state = "bluefrag"
+	inhand_icon_state = "flashbang"
+	resistance_flags = FIRE_PROOF
+
+
+/obj/item/grenade/gas_crystal/healium_crystal
+	name = "Healium crystal"
+	desc = "A crystal made from the Healium gas, it's cold to the touch."
+	icon_state = "healium_crystal"
+	///Amount of stamina damage mobs will take if in range
+	var/stamina_damage = 30
+	///Range of the grenade that will cool down and affect mobs
+	var/freeze_range = 4
+	///Amount of gas released if the state is optimal
+	var/gas_amount = 250
+
+/obj/item/grenade/gas_crystal/proto_nitrate_crystal
+	name = "Proto Nitrate crystal"
+	desc = "A crystal made from the Proto Nitrate gas, you can see the liquid gases inside."
+	icon_state = "proto_nitrate_crystal"
```

Comment:
Too lazy to rename the icon but I would like to point out that it could be changed to "pluonium_crystal" if you so choose